Assets/Scripts/Menus/SettingsMenu.cs

```diff
@@ -34,6 +55,18 @@ public void SetSFXVolume(float sfxVol)
         PlayerPrefs.SetFloat("SFXVolume", sfxVol); // Sets the value of SliderVolume to the sound effect volume value.
         SFXMixer.SetFloat("SFXVolume", Mathf.Log10(sfxVol) * 20);
 
+        if (sfxVol <= 0)
+            SFXMixer.SetFloat("SFXVolume", 0);
+
+        PlayerPrefs.Save();
+    }
+
+    public void SetMouseSens(float _sens)
```

Comment:
no harm in naming this `SetMouseSensitivity`. We aren't going to run out of characters and it makes this function's purpose much clearer.

Reply:
Equally, the parameter can be named `sensitivity`

Reply:
As long as the short hand is understandable, I think it's fine to abbreviate.